README.md

```diff
@@ -6,58 +6,62 @@
 [![Latest Release](https://img.shields.io/github/v/release/bitovi/bitops)](https://github.com/bitovi/bitops/releases)
 [![Join our Slack](https://img.shields.io/badge/slack-join%20chat-611f69.svg)](https://www.bitovi.com/community/slack?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-BitOps is a way to describe the infrastructure and things deployed onto that infrastructure for multiple environments in a single place called an [Operations Repo](docs/operations-repo-structure.md).
-
-https://bitovi.github.io/bitops/
+### tl;dr
+BitOps is two things; 
+- A automated tool [orchestrator](docs/about.md)
+- A way to describe [infrastructure](docs/operations-repo-structure.md)
 
 ---------------------
 
 ## Features
 
-* **[Configurable](docs/configuration-base.md):** Configure how you want bitops to deploy your application with environment variables or yaml
-* **[Event Hooks](docs/lifecycle.md):** If bitops doesn't have built-in support for your usecase, execute arbitrary bash scripts at different points in bitops' lifecycle.
+* **[Configurable](docs/configuration-base.md):** Configure how you want bitops to deploy your application with yaml or environment variables.
+* **[Event Hooks](docs/lifecycle.md):** If bitops doesn't have built-in support for your usecase, execute arbitrary bash scripts at different points using bitops' lifecycle.
 * **[Pipeline Agnostic](docs/examples.md):** By bundling all logic in bitops, you can have the same experience regardless of which pipeline service runs your CI. You can even run bitops locally!
+* **[Customizable](docs/plugins.md):** Configure how what tools you want installed in your BitOps image. Only take what you need, leave the bloat behind. 
 
 ## How it works
 
-BitOps is a boiler plate docker image for DevOps work. An operations repository is mounted to a bitops image's `/opt/bitops_deployment` directory. BitOps will
+BitOps is a tool orchestrator packaged in a docker image for DevOps work. An operations repository is mounted to a bitops image's `/opt/bitops_deployment` directory. BitOps will parse through the operations repo and;
 
-* Auto-detect any configuration belonging to one of its [supported tools](#supported-tools)
+* Auto-detect bitops configuration files within tool directories
 * Loop through each tool and
+  * Read in `yaml` configuration
   * Run any pre-execute hooks
-  * Read in `yml` configuration
   * Execute the tool
   * Run any post-execute hooks
 
-## Run BitOps
+## Quick Start
 BitOps is packaged as a docker image and is available on [dockerhub](https://hub.docker.com/r/bitovi/bitops).
 ```
-docker pull bitovi/bitops
+docker pull bitovi/bitops:2.0.0-latest
 cd $YOUR_OPERATIONS_REPO
-docker run bitovi/bitops -v .:/opt/bitops_deployment
+docker run bitovi/bitops:2.0.0-latest -v .:/opt/bitops_deployment
 ```
 
-## Configure BitOps
+Need a hand? We got you! Check out our [Example Operation Repos](https://github.com/bitovi/operations-test)
```

Comment:
break this out into 2 parts:
1. want examples?
2. need a hand? -> [Bitovi DevOps Consulting](https://www.bitovi.com/devops-consulting) or hit us up on the Bitovi Community Slack (link) in the `#bitops` channel